docs/guides/dev_guide.rst

```diff
@@ -35,13 +35,20 @@ Contributing
 * Please keep large lint changes separate from new features, though this point
   should become less relevant over time.
 * All pull requests should be squashed down to logical blocks of distinctive
-  functionality that work by themselves and do not result in brokenness of master
+  functionality that work by themselves and do not result in brokenness of master.
 
   * As an example, if you were working on a test which required new pages,
     utilities and tests, it would be OK to split the page, utility and test
     changes into separate requests or commits, providing they were in the correct
     order of dependency.
 
+* All commits should be signed.
```

Comment:
**REQUIRED:** GPG signing is good practice, however the only thing it is required for is PRT testing.

Reply:
I added as "should" because we will always have those 2 errors (ci/downstream-58z and ci/downstream-59z). They are also difficult to understand and sound like a problem. Are you sure I should change this text? I was thinking about update the text to mention the 2 errors somehow.

Reply:
I noticed that the checkings were removed. Is this just a temporary change?

Reply:
Done. But I added more information about the checks. Could you please review again? I can remove the new info if you prefer.